sonder/utils.py

```diff
@@ -10,3 +10,25 @@ def pgn_to_uci(pgn):
     for move in game.mainline_moves():
         moves.append(move.uci())
     return moves
+
+def import_pgn_to_db(pgn):
+    pgn_in = open(pgn)
+    while True:
+        game = chess.pgn.read_game(pgn_in)
+        if game is None:
+            break
+        into_db(game)
+
+def into_db(game):
+    from sonder.analysis.models import Game, Player
```

Comment:
This can be moved to the top level imports in this file. We only had to do this in the bin files.